src/Coinbase/Coinbase.jl

```diff
@@ -68,7 +68,15 @@ function Base.show(io::IO, e::CoinbaseAPIError)
     return print(io, "message = ", "\"", e.message)
 end
 
-function CryptoAPIs.request_sign!(::CoinbaseClient, query::Q, ::String)::Q where {Q<:CoinbaseCommonQuery}
+function CryptoAPIs.request_sign!(::CoinbaseClient, query::Q, ::String)::Q where {Q<:CoinbasePublicQuery}
+    return query
+end
+
+function CryptoAPIs.request_sign!(client::CoinbaseClient, query::Q, ::String)::Q where {Q<:CoinbasePrivateQuery}
+    query.timestamp = string(round(Int64, datetime2unix(Dates.now(UTC))))
+    query.signature = nothing
+    message = join([query.timestamp, method, endpoint, query])
```

Comment:
`method` and `endpoint` variables are not declared anywhere in advance

Reply:
you can very easily test the signature

```julia
using CryptoAPIs
using CryptoAPIs.Binance

my_client = BinanceClient(;
    base_url = "https://api.binance.com",
    public_key = "my_public_key",
    secret_key = "my_secret_key",
)

my_query = Binance.Spot.AccountTradeQuery(; symbol = "ADAUSDT")

my_endpoint = "api/v3/myTrades"

CryptoAPIs.request_sign!(my_client, my_query, my_endpoint)

julia> my_query.signature
"243a6d36ddbd109b598cb0cd6e2810f16423edf8e66323ed6cf4238412c7bdde"
```